amr-wind/core/FieldBCOps.H

```diff
@@ -169,9 +186,10 @@ struct DirichletOp
                 }
 
                 // Check if the point in question is on the boundary
+                int mod_bigEnd = domain_box.bigEnd(idir) + IdxOp(idir);
```

Comment:
`const` and maybe a more descriptive name? `bigEnd_depending_on_if_you_are_cc_or_fc` ;-) obv not that either.